src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -270,7 +270,7 @@ public void testSearchFacetsDistribution() throws Exception {
 
         SearchResult searchResult = index.search(searchRequest);
 
-        assertEquals(3, searchResult.getHits().size());
+        assertEquals(1, searchResult.getHits().size());
```

Comment:
Why do you receive fewer hits?

Reply:
I think it's because of the `typo tolerance` feature.
I was receiving 3 results for the query `knight` including 2 with `night`
Now I receive only 1 result with `knight`

Reply:
Do you think it's not an expected behavior?

Reply:
I'm not sure, could you open an issue in `meilisearch/meilisearch` ? Since we are removing acceptable candidates

Reply:
It is an expected behavior 😅 https://github.com/meilisearch/meilisearch/issues/2348